src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+function createPromise() {
```

Comment:
There is no reason to create this function, as we have two different promises and this logic doesn't repeat. Usually, it is good to separate logic but in this case, it makes no sense, because we use it only once